src/fftvis/cpu/cpu_simulate.py

```diff
@@ -110,7 +111,9 @@ def simulate(
             "CoordinateRotationAstropy", "CoordinateRotationERFA"
         ] = "CoordinateRotationERFA",
         coord_method_params: dict | None = None,
+        skew_to_cartesian: np.ndarray | None = None,
```

Comment:
Did we add docstrings for these into the base class?

Reply:
Removed this in favor of a more general gridding function that finds a set of basis vectors for the basis transformation.